src/tcp_cubic_search.c

```diff
@@ -433,8 +465,23 @@ static void bictcp_state(struct sock *sk, u8 new_state)
 {
 	if (new_state == TCP_CA_Loss) {
 		bictcp_reset(inet_csk_ca(sk));
-		bictcp_hystart_reset(sk);
+
+		//////////////////////// SEARCH ////////////////////////
+		switch (ca->ss_exit_point) {
+			case SS_EXIT_POINT_SEARCH:
+				bictcp_search_reset(ca);  // Reset SEARCH slow start
+				break;
+			case SS_EXIT_POINT_HYSTART:
+				bictcp_hystart_reset(sk);  // Reset HyStart slow start
+				break;
+			case SS_EXIT_POINT_NONE:
```

Comment:
it is fine to remove line 477 and line 478. or we can still use if statement here.